### PR TITLE
switched and aligned download button

### LINKED ISSUE #226: BUG: Icon Fixes
image

- Switch the order of the download and trash icon
- Move the download icon to the end of the container, it should align with the right edge of the participants component

### PR DESCRIPTION
**Pull request by sharansingh1**
image

### CODE REVIEW

**Review by rfairooz — DISMISSED** [review]
The margin you have right now is not completely aligned to the right. Just have to increase the margin. Make sure you test your code out on full screen

File: src/components/Admin/Toolbar.jsx [hunk 1]
```diff
@@ -135,6 +129,12 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
           className="ml-5 text-hackathon-darkgray hover:opacity-70 duration-150"
         />
       </button>
+      <button className="ml-auto">
+        <FaDownload
+          size={22.5}
+          className="text-hackathon-darkgray hover:opacity-70 duration-150 "
```
Comment (shahdivyank) [inline src/components/Admin/Toolbar.jsx]:
```suggestion
          className="text-hackathon-darkgray hover:opacity-70 duration-150"
```

File: src/components/Admin/Toolbar.jsx [hunk 2]
```diff
@@ -135,6 +129,12 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
           className="ml-5 text-hackathon-darkgray hover:opacity-70 duration-150"
         />
       </button>
+      <button className="ml-auto">
```
Comment (shahdivyank) [inline src/components/Admin/Toolbar.jsx]: remove the ml-auto, we should be able to determine the margin left without auto

Comment (sharansingh1) [reply src/components/Admin/Toolbar.jsx]: i found that making the margin to be 600px also works, can I do that?

Comment (shahdivyank) [reply src/components/Admin/Toolbar.jsx]: Find a way to do it without such a large margin

File: src/components/Admin/Toolbar.jsx [hunk 3]
```diff
@@ -117,33 +117,38 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
...
+          <Popup
+            buttonColor="red"
+            text={popup}
+            callBack={handleDelete}
+            setText={setPopup}
+            title="Delete Confirmation"
+          />
+        )}
+      </div>
+      <div>
```
Comment (shahdivyank) [inline src/components/Admin/Toolbar.jsx]: remove, this div isn't doing anything

File: src/components/Admin/Toolbar.jsx [hunk 4]
```diff
@@ -117,33 +117,38 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
...
+        <button>
+          <FaDownload
+            size={22.5}
+            className=" text-hackathon-darkgray hover:opacity-70 duration-150 "
```
Comment (shahdivyank) [inline src/components/Admin/Toolbar.jsx]:
```suggestion
            className=" text-hackathon-darkgray hover:opacity-70 duration-150"
```

File: src/components/Admin/Toolbar.jsx [hunk 5]
```diff
@@ -117,33 +117,38 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
...
+            setText={setPopup}
+            title="Delete Confirmation"
+          />
+        )}
+      </div>
+      <div>
+        <button>
+          <FaDownload
+            size={22.5}
+            className=" text-hackathon-darkgray hover:opacity-70 duration-150"
```
Comment (shahdivyank) [inline src/components/Admin/Toolbar.jsx]:
```suggestion
            className="text-hackathon-darkgray hover:opacity-70 duration-150"
```

**Pull request closed without merging**